lib/handlers/passwordReset/SAResetV1.js

```diff
@@ -0,0 +1,23 @@
+const {getEmailAdapter} = require('./../../infrastructure/email');
+
+const process = async (config, logger, data) => {
+  const email = getEmailAdapter(config, logger);
+  await email.send(data.email, 'sa-reset-password', {
+    firstName: data.firstName,
+    lastName: data.lastName,
+    returnUrl: `${config.notifications.profileUrl}/register/${data.invitationId}`,
```

Comment:
You will not have an invitation id at this point. To be able to put them directly into the merge process we would need to send them to the service they started from.